app/src/main/java/hu/bme/aut/fitary/dataSource/FirebaseDataSource.kt

```diff
@@ -48,7 +53,7 @@ class FirebaseDataSource @Inject constructor(
     private fun mapWorkoutExercisesToDomain(workout: Workout): MutableList<DomainExercise> {
         val domainExercises = mutableListOf<DomainExercise>()
 
-        for (i in 1 until workout.exercises.size) {
```

Comment:
This bug was especially hard to spot and it took a long time to find out that every workout missed it's first exercise.